srcpkgs/python3-gnupg/template

```diff
@@ -1,18 +1,23 @@
 # Template file for 'python3-gnupg'
 pkgname=python3-gnupg
 _pkgname=python-gnupg
-version=0.4.6
-revision=4
+version=0.4.8
+revision=1
 wrksrc=$_pkgname-$version
 build_style=python3-module
 hostmakedepends="python3-setuptools"
+checkdepends="gnupg"
 depends="python3 gnupg"
 short_desc="Python3 wrapper for GnuPG"
 maintainer="Lon Willett <xgit@lonw.net>"
 license="BSD-3-Clause"
-homepage="https://bitbucket.org/vinay.sajip/python-gnupg"
+homepage="https://github.com/vsajip/python-gnupg"
 distfiles="${PYPI_SITE}/p/${_pkgname}/${_pkgname}-${version}.tar.gz"
-checksum=3aa0884b3bd414652c2385b9df39e7b87272c2eca1b8fcc3089bc9e58652019a
+checksum=b64de1ae5cedf872b437201a566fa2c62ce0c95ea2e30177eb53aee1258507d7
+
+pre_check() {
+	export NO_EXTERNAL_TESTS=1
+}
```

Comment:
These tests require stuff manually set up by the developer on their dev machine. It's disabled in CI by upstream as well, using this env var.

Reply:
Done